tests/slow/src/test/scala/tests/feature/Ammonite213Suite.scala

```diff
@@ -0,0 +1,5 @@
+package tests.feature
+
+import scala.meta.internal.metals.{BuildInfo => V}
+
+class Ammonite213Suite extends tests.BaseAmmoniteSuite(V.scala213)
```

Comment:
Honestly, I would just keep them all in the same file.

Reply:
Done.